consensus/types/src/chain_spec.rs

```diff
@@ -776,6 +779,9 @@ pub struct Config {
     // TODO(merge): remove this default
     #[serde(default = "default_terminal_block_hash_activation_epoch")]
     pub terminal_block_hash_activation_epoch: Epoch,
+    // TODO(merge): remove this default
```

Comment:
I'm not exactly sure what we had in mind to replace this so I made it like all the rest. If someone wants to fill me in I can do this.

Reply:
I think it's OK for now. The fix I had in mind was to `superstruct` the `Config` structs so that we have different configs for different hard forks.